lib/ClaimVerifier.ts

```diff
@@ -86,5 +95,21 @@ export class ClaimVerifier {
     }
     return null
   }
+  
+  private async verifySignature(issuer: string, issuerClaims: Claim[]) : Promise<boolean>{
+    const didDocument = await this.getDidDocument(issuer);
+    const authenticationClaimVerifier = new AuthTokenVerifier(didDocument);
+    issuerClaims.map(async (claim) => {
+      if (claim.iss !== issuer){
+        return false
+      }
+      const claimToken = claim.issuedToken as string
+      const verifiedIssuer = await authenticationClaimVerifier.verify(claimToken)
+      if (verifiedIssuer !== issuer){
+        return false
+      }
+    })
+    return true
```

Comment:
I think this `return false` might not be quite right. Isn't it return false from the function in the map, not false for the `verifySignature` function?